src/redisearch_rs/inverted_index_bencher/Cargo.toml

```diff
@@ -26,6 +26,10 @@ ffi.workspace = true
 inverted_index.workspace = true
 itertools.workspace = true
 redis_mock.workspace = true
+varint_ffi = { path = "../c_entrypoint/varint_ffi" }
+
+[dev-dependencies]
+varint_ffi = { path = "../c_entrypoint/varint_ffi" }
```

Comment:
Why is the extra dev-dependency one needed?

Reply:
To ensure, this symbol used by the C code is there, see

```rust
    // The encode C implementation relies on this symbol. Re-export it to ensure it is not discarded by the linker.
    #[allow(unused_imports)]
    pub use varint_ffi::WriteVarintFieldMask;
```

Reply:
Is the symbol defined in a test or something? I'm confused why it won't be there with just the `[dependencies]` definition